tests/code-splitting/subtests/dynamic-import-preloading/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Dynamic import Preloading
+importance: 1
+shortDesc: 'Can code splitting be preloaded?'
+---
+
+# Introduction
+
+When creating split points [using dynamic import](/code-splitting/dynamic-import/), it is possible to construct a "waterfall" scenario where JavaScript bundles depend on additional bundles discovered once executed. The ability to preload nested dependency bundles is important in these cases, since doing so can allow the dependency bundles to be downloaded and evaluated in parallel.
+
+# The Test
+
+This test checks to see if it's possible to request code-splitted dependency bundles before the consuming bundle has been loaded, either via [preload](https://web.dev/preload-critical-assets/) or async script loading.
+
+**index.js**
+
+```js
+setTimeout(async () => {
+  const { shout } =
+    Math.random() < 0.5
```

Comment:
I can’t find an explanation in this README for this `random()`? What is this for?

Reply:
I proposed that, but you can also use other conditions. In the end you need to `import()` two different modules that share a common dependency. This way you create a chunking where one output depends on another one that can be loaded in parallel or preloaded.

Reply:
OOoh, it’s just so we have 2 modules with a shared dependency. Got it. It warrants an explanation or even simpler: just 2 consecutive `import()` calls.

Reply:
That would also work. I proposed that to simulate an A/B test scenario.

Reply:
That makes sense. A short comment to explain this would be good, tho. (cc @AjayPoshak)